Project/Assets/Prairie/Framework/Script/Interaction/AnnotationInteraction.cs

```diff
@@ -169,7 +179,11 @@ void Update()
                 Cursor.visible = false;
                 Cursor.lockState = CursorLockMode.Locked;
                 
-				this.SetPlayerIsFrozen (false);
+				FirstPersonInteractor player = this.GetPlayer ();
```

Comment:
This is the second appearance of these lines of code. Would it be better to extract this into a function that takes a boolean parameter?

Not quite sure what it would be called though...  at worst, could do`setPlayerCanMoveAndDrawGui(bool)` or something.